Nodejs/Product/Nodejs/NodejsPackage.cs

```diff
@@ -182,6 +183,10 @@ internal sealed partial class NodejsPackage : CommonPackage {
         // Overridden Package Implementation
         #region Package Members
 
+        // Hold references for the subscribed events. Otherwise the callbacks will be garbage collected
+        // after the initialization
+        private List<EnvDTE.CommandEvents> subscribedCommandEvents = new List<EnvDTE.CommandEvents>();

```

Comment:
private members should begin with underscores. Also move this to the top with the rest.